### PR TITLE
ru: translate sentence to Russian

### PR DESCRIPTION
**Pull request by Abrasha**

### CODE REVIEW

File: README-ru.md [hunk 1]
```diff
@@ -151,7 +151,7 @@
 
 - Используйте `man ascii` для просмотра хорошей ASCII таблицы, с шестнадцатеричными и десятичными значениями. Для информации по основным кодировкам полезны: `man unicode`, `man utf-8` и `man latin1`.
 
-- Используйте `screen` или [`tmux`](https://tmux.github.io/) для того, чтобы иметь несколько экранов в одном терминале. Это особенно полезно, когда вы работаете с удаленным сервером по ssh, тогда вы можете подключаться/отключаться от сессий. `byobu` can enhance screen or tmux providing more information and easier management. Более минималистичный подход для этого – использование [`dtach`](https://github.com/bogner/dtach).
+- Используйте `screen` или [`tmux`](https://tmux.github.io/) для того, чтобы иметь несколько экранов в одном терминале. Это особенно полезно, когда вы работаете с удаленным сервером по ssh, тогда вы можете подключаться/отключаться от сессий. `byobu` может улучшить использование `screen` или `tmux` предоставляя больше информации и удобное управление. Более минималистичный подход для этого – использование [`dtach`](https://github.com/bogner/dtach).
```
Comment (idvoretskyi) [inline README-ru.md]: A comma is needed after `tmux`, before "предоставляя".

Comment (Abrasha) [inline README-ru.md]: @idvoretskyi yes, thanks. I will update PR

Comment (Abrasha) [inline README-ru.md]: @idvoretskyi I have added comma. Any other suggestions?

Comment (idvoretskyi) [inline README-ru.md]: @Abrasha no, LGTM!

**Comment by jlevy** [comment]
@olegberman review?

**Comment by olegberman** [comment]
This can absolutely go through! 👍 Thanks!

**Pull request merged**